### PR TITLE
SimLex test - set creation

### PR DESCRIPTION
**Pull request by Eszti**
`scripts/add_wordpair_to_simlex_test.py WORD1 WORD2` adds the definitions of the two words to input xml file

### CODE REVIEW

File: conf/default.cfg [hunk 1]
```diff
@@ -118,7 +118,7 @@ lang = en
 
 #options for testing the word similarity module
 [word_sim]
-4langpath = /home/recski/sandbox/4lang
+4langpath = /home/eszter/projects/4lang
```
Comment (recski) [inline conf/default.cfg]: This line should not be part of the commit - it'll cause modules to stop working on our other servers. The environment variable FOURLANGPATH will be a better solution, of course...

**Comment by Eszti** [comment]
I corrected my mistake. Is it ok this way now, or should I open a new pull request?
